_includes/navbar.ext

```diff
@@ -58,10 +58,8 @@
                 <ul class="dropdown-menu" aria-labelledby="projects_menu">
                   <li><a href="{{ site.baseurl }}/projects/affiliated.html">HSF Affiliated Projects and Software</a></li>
                   <li><a href="{{ site.baseurl }}/projects/guidelines.html">Projects and Software Guidelines</a></li>
-                  <li class="divider"></li>
-                  <li><a href="{{ site.baseurl }}/project_guidelines.html">How to join as a project</a></li>
-                  <li><a href="{{ site.baseurl }}/projects.html">Member Projects</a></li>
-                  <li><a href="{{ site.baseurl }}/services.html">Development Services</a></li>
+                  <li><a href="{{ site.baseurl }}/projects/projects.html">Member Projects</a></li>
+                  <li><a href="{{ site.baseurl }}/projects/badges.html">Affiliate Badges</a></li>
```

Comment:
So I understand you will create a new page here?

Reply:
It was created already, it just hadn't been linked from the main menu (it was just an internal link from one of the other pages).